cmd/create/cluster/cmd.go

```diff
@@ -355,6 +373,55 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Subnet IDs
+	awsClient, err := aws.NewClient().
+		Region(region).
+		Logger(logger).
+		Build()
+	if err != nil {
+		reporter.Errorf("Failed to create awsClient: %s", err)
+		os.Exit(1)
+	}
+
+	subnetIDs := args.subnetIDs
+	subnets, err := awsClient.GetSubnetIDs()
+	mapSubnetToAZ := make(map[string]string)
+	mapAZCreated := make(map[string]bool)
+	options := make([]string, len(subnets))
+	for i, subnet := range subnets {
+		subnetID := awssdk.StringValue(subnet.SubnetId)
+		availabilityZone := awssdk.StringValue(subnet.AvailabilityZone)
+
+		// Create the options to prompt the user.
+		options[i] = setSubnetOption(subnetID, availabilityZone)
+		mapSubnetToAZ[subnetID] = availabilityZone
+		mapAZCreated[availabilityZone] = false
+	}
+	if interactive.Enabled() && (!multiAZ || len(mapAZCreated) >= 3) {
```

Comment:
I think we still need to add `len(options) > 0` to this condition check

Reply:
Done.